test/commands/cm/cm.withdraw.test.ts

```
@@ -0,0 +1,70 @@
+        await new Promise(resolve => setTimeout(resolve, 10000))
+    })
+
+    it('can withdraw from a candy machine', async () => {
+        const cmName = "testCm3"
+
+        try {
+            const { collectionId } = await createCoreCollection()
+
+            // console.log('Creating test candy machine directory')
+            // Await the directory creation
+            await execAsync("npm run create-test-cm -- --name=" + cmName + " --with-config --collection=" + collectionId)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)